### PR TITLE
update test matrix

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed-off-by: pbanasiak <przemyslaw.banasiak@3mdeb.com>

### CODE REVIEW

File: docs/variants/talos_2/test-matrix.md [hunk 1]
```diff
@@ -9,14 +9,17 @@ subjected from before the release of the new binary.
 
 | No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
 |:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
-| 1.   | [coreboot base port][CBP]                         | CBP           | CBP001.001, CBP002.001, CBP004.001   |
+| 1.   | [Coreboot base port][CBP]                         | CBP           | CBP001.001, CBP002.001, CBP004.001   |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/test-matrix.md:12]: `coreboot` should always be written in lower case.

File: docs/variants/talos_2/test-matrix.md [hunk 2]
```diff
@@ -26,12 +29,15 @@ subjected from before the release of the new binary.
...
+[USB]: ../../unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support/
+[NVM]: ../../unified-test-documentation/dasharo-compatibility/312-nvme-support/
+[DSP]: ../../unified-test-documentation/dasharo-compatibility/31E-display-ports-and-lcd/
 
 ## Module: Dasharo performance
 
 | No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
 |:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
-| 1.   | [coreboot bring up time measurement][CBMEM]       | CBMEM         | All                                  |
+| 1.   | [Coreboot bring up time measurement][CBMEM]       | CBMEM         | All                                  |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/test-matrix.md:40]: As above

File: docs/variants/talos_2/test-matrix.md (hunk 1 above)
Comment (PrzemyslawBanasiak13) [reply docs/variants/talos_2/test-matrix.md]: fixed

File: docs/variants/talos_2/test-matrix.md (hunk 2 above)
Comment (PrzemyslawBanasiak13) [reply docs/variants/talos_2/test-matrix.md]: fixed

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**